### ISSUE TITLE
Problem with ruby & ncurses

### ISSUE THREAD

**Issue by phineas0fog** [cbbrowne/tpp#9]
Hello :)
I want to try tpp but when I use `tpp example.tpp` the first slide is displayed but there is this message at the bot of the term : 
`ruby: symbol lookup error: /usr/lib/ruby/vendor_ruby/2.4.0/x86_64-linux/ncurses_bin.so: undefined symbol: rb_thread_select`
And my term crash...
Here is the file: [example.tpp](https://gist.github.com/phineas0fog/cd76f8211433fb1cc6994855e851da37)
Thank you for the s olution :D

**Comment by jamesalbert** [comment]
http://stackoverflow.com/a/32372940/1701961

**Comment by phineas0fog** [comment]
I have the version 1.3.1-10 of ruby-ncurses (I use arch). I try to reinstall it but there is the same problem...

**Comment by jamesalbert** [comment]
Yeah but you're using ruby 2.4.0 which looks like it doesn't support `rb_thread_select`. I'm no ruby expert, but from the look of the errors, I would guess that you need to either downgrade ruby or upgrade ncurses. Again, no ruby expert and I haven't dug into this. Just got an email alert so I thought I'd help if I could
